packages/migrate/src/helpers/transform-utils.ts

```diff
@@ -12,16 +12,31 @@ export function getInterfaceByName(
   return matched as ts.InterfaceDeclaration;
 }
 
+export function hasNameInImportClause(
+  importClause: ts.ImportClause | undefined,
+  importName: string
+): boolean {
+  if (importClause?.name?.getFullText().trim() === importName) {
+    return true;
+  }
+  if (
+    importClause?.namedBindings &&
+    (importClause?.namedBindings as ts.NamedImports).elements.find(
+      (e) => e.name.getFullText().trim() === importName
+    )
+  ) {
+    return true;
+  }
+
+  return false;
+}
+
 export function getImportByName(
   sourceFile: ts.SourceFile,
   typeName: string
 ): ts.ImportDeclaration | undefined {
   const matched = sourceFile.statements.find(
-    (s) =>
-      ts.isImportDeclaration(s) &&
-      (s.importClause?.namedBindings as ts.NamedImports).elements.find(
-        (e) => e.name.getFullText().trim() === typeName
-      )
+    (s) => ts.isImportDeclaration(s) && hasNameInImportClause(s.importClause, typeName)
   );
   return matched as ts.ImportDeclaration;
```

Comment:
Will this fail if `matched` is undefined?

Reply:
No. The return type of this function is specified as ts.ImportDeclaration | undefined. The function that calls this function handles both.